pl_bolts/datasets/emnist_dataset.py

```diff
@@ -14,9 +15,16 @@
     warn_missing_pkg("PIL", pypi_name="Pillow")
 
 
-@under_review()
 class BinaryEMNIST(EMNIST):
-    def __getitem__(self, idx):
+    """Binarized EMNIST Dataset.
+
+    EMNIST dataset binarized using a thresholding operation. Threshold is set to 127. Note that the images are binarized
+    prior to the application of any transforms.
+    """
+
+    threshold = 127.0
```

Comment:
Can this threshold be an argument to `__init__`?

Reply:
Added threshold as an optional argument with default value as 127.